plugins/net_plugin/net_plugin.cpp

```diff
@@ -475,6 +476,7 @@ namespace eosio {
       std::function<void()> increment_dropped_trxs;
       
    private:
+      static const std::map<std::string, int> prefix_multipliers;
```

Comment:
This can be:
```
      inline static const std::map<std::string, int> prefix_multipliers{
              {"",1},{"K",pow(10,3)},{"M",pow(10,6)},{"G",pow(10, 9)},{"T",pow(10, 12)},
              {"Ki",pow(2,10)},{"Mi",pow(2,20)},{"Gi",pow(2,30)},{"Ti",pow(2,40)}};
```

Reply:
Moved, and type changed to prevent overflows.